### PR TITLE
Changes for Zeek to build as a subproject

### PR DESCRIPTION
**Pull request by sethhall**
This is some changes to cmake buildable as a subproject by changing the variables that are used.

Several of the subprojects updated branches as well and need to be merged too.

### CODE REVIEW

File: src/zeek-setup.cc [hunk 1]
```diff
@@ -549,6 +549,7 @@ SetupResult setup(int argc, char** argv, Options* zopts)
 		add_input_file("base/init-default.zeek");
 
 	add_input_file("builtin-plugins/__load__.zeek");
+	add_input_file("builtin-plugins/__preload__.zeek");
```
Comment (0xxon) [inline src/zeek-setup.cc:552]: this is unrelated and part of https://github.com/zeek/zeek/pull/1615/files, right? If yes - it should probably not be part of this PR.

Comment (sethhall) [reply src/zeek-setup.cc:552]: Whoops, right.  That was in there because the work I was doing with this needed both fixes but I accidentally let this slip in.

**Review by rsmmr — APPROVED** [review]
Looks good to me, afaict the changes shouldn't change anything for a normal Zeek build.

I am seeing some inconsistent whitespace in the submodule changes; can't flag them because GH doesn't seem let me add comments to changes in submodules.
